kythe/typescript/BUILD

```diff
@@ -4,6 +4,17 @@ load("@aspect_rules_js//js:defs.bzl", "js_test")
 
 package(default_visibility = ["//kythe:default_visibility"])
 
+# If you need to add / update dependencies:
+#
+# 0. Ensure you have a recent nodejs and pnpm installed.
+# 1. Edit package.json in this directory as appropriate
+# 2. From the *root dir* of the project, run
+#
+#       cp kythe/typescript/package.json .
+#       pnpm import
```

Comment:
You should also be able to use the bazel-hosted pnpm, but I can't remember the incantation.